src/PJ_sch.c

```diff
@@ -0,0 +1,206 @@
+/*
+** Author : Piyush Agram
+** This code is in the public domain and implements the SCH coordinate system first developed at JPL.
```

Comment:
See how FrankW dealt with similar situation in GDAL : https://github.com/OSGeo/gdal/blob/trunk/gdal/frmts/pds/pdsdataset.cpp